sorc/global_cycle.fd/cycle.f90

```diff
@@ -516,19 +521,38 @@ SUBROUTINE SFCDRV(LUGB, IDIM,JDIM,LSM,LENSFC,LSOIL,DELTSFC,  &
 !--------------------------------------------------------------------------------
 
  IF (DO_SFCCYCLE) THEN
+   ALLOCATE(SLMASKL(LENSFC), SLMASKW(LENSFC))
+! for running uncoupled (non-fractional grid)
+   DO I=1,LENSFC
+     IF(NINT(SLMASK(I)) == 1) THEN
+       SLMASKL(I) = 1.0_KIND_io8
```

Comment:
Could you add a short name for what slmask, slmaskl & slmaskw stand for?

Reply:
Done at ab52c26.